Mathlib/Computability/TuringMachine.lean

```diff
@@ -874,88 +853,79 @@ theorem exists_enc_dec [Inhabited Γ] [Finite Γ] :
 
 variable {Λ σ : Type*}
 
-local notation "Stmt₁" => Stmt Γ Λ σ
-
-local notation "Cfg₁" => Cfg Γ Λ σ
-
 /-- The configuration state of the TM. -/
 inductive Λ'
   | normal : Λ → Λ'
-  | write : Γ → Stmt₁ → Λ'
-
-local notation "Λ'₁" => @Λ' Γ Λ σ  -- Porting note (https://github.com/leanprover-community/mathlib4/issues/10750): added this to clean up types.
+  | write : Γ → Stmt Γ Λ σ → Λ'
 
-instance [Inhabited Λ] : Inhabited Λ'₁ :=
+instance [Inhabited Λ] : Inhabited (@Λ' Γ Λ σ) :=
```

Comment:
Can we make the arguments explicit, and remove the `@`?

Reply:
Done.